src/frontend/src/optimizer/plan_node/generic/agg.rs

```diff
@@ -443,7 +451,7 @@ impl<PlanRef: stream::StreamPlanRef> Agg<PlanRef> {
                     }
                 }
                 // TODO: is its state a Table?
-                AggKind::BitAnd | AggKind::BitOr => {
+                AggKind::BitAnd | AggKind::BitOr | AggKind::BoolAnd | AggKind::BoolOr => {
```

Comment:
Looks like these agg calls with non-append-only input should have `MaterializedInputState`

Reply:
we can do it better by just storing the count of `1`  in each bit. I am not sure which one is better.
1. rewrite it to multiple `count` in the optimizer, but the pre and post Project needs some bitwise expression and maybe we even need a BIT datatype for better support. 
2. build a new `AggCallState::Table` state for it just like `AggKind::ApproxCountDistinct`
Never mind, I think they(bitAnd and bitOr) are not required now and are less important than the bit_xor(which is often used for verification), so we can design and implement it later.